test/src/touchscreen.spec.ts

```diff
@@ -51,4 +53,68 @@ describe('Touchscreen', function () {
       })
     ).toEqual(['Touchstart: 0', 'Touchend: 0']);
   });
+
+  it('should report press', async () => {
+    const {page, server} = getTestState();
+    const iPhone = KnownDevices['iPhone 6']!;
+    await page.setViewport({
+      width: KnownDevices['iPhone 6'].viewport.width,
+      height: KnownDevices['iPhone 6'].viewport.height,
+      isMobile: true,
+      hasTouch: true,
+    });
+    await page.emulate(iPhone);
+    await page.goto(server.PREFIX + '/input/touches-press.html');
+    await page.touchscreen.press(100, 100, 1000);
```

Comment:
let's not relay on timing and especially such a long timeouts in the tests

Reply:
let's try using https://sinonjs.org/releases/latest/fake-timers/ (we already have sinon as a dependency)

Reply:
I do it tomorrw